packages/ozone/src/api/moderator/addUser.ts

```diff
@@ -0,0 +1,41 @@
+import { AuthRequiredError, InvalidRequestError } from '@atproto/xrpc-server'
+import { Server } from '../../lexicon'
+import AppContext from '../../context'
+import { getUserRole } from '../util'
+
+export default function (server: Server, ctx: AppContext) {
+  server.tools.ozone.moderator.addUser({
+    auth: ctx.authVerifier.moderator,
```

Comment:
```suggestion
    auth: ctx.authVerifier.modOrAdminToken,
```

Reply:
we try to keep track of requester's DID in `lastUpdatedBy` but with admin tokens, we don't get a DID. wondering if we should just store something like `admin_token` in `lastUpdatedBy` when these requests are made with admin token?

Reply:
Ah interesting!  Yeah, that could work.